src/main/kotlin/lotto/view/InputView.kt

```diff
@@ -0,0 +1,28 @@
+package lotto.view
+
+import lotto.model.Money
+import lotto.model.WinningNumbers
+
+class InputView {
```

Comment:
코틀린은 객체 선언 기능을 통해 싱글턴 패턴을 언어에서 기본 지원합니다. 아래의 글이 도움 될 거예요.
https://kotlinlang.org/docs/object-declarations.html#object-declarations

```suggestion
object InputView {
```